### PR TITLE
[1]: Adds and implements Nunito font family

### PR DESCRIPTION
**Pull request by comountainclimber**
Takes care of https://github.com/comountainclimber/rn-task-manager/issues/1

Although the design spec is using `Avenir` it doesnt appear as though this font family is free and open source, so I chose the next best option - `Nunito`. More about this font can be found [here](https://fonts.google.com/specimen/Nunito)

I am leveraging the `expo-font` package to asynchronously load our fonts while preventing an application render until font load is complete. Essentially taken directly from the expo documentation which can be found [here](https://docs.expo.io/versions/latest/guides/using-custom-fonts/).

This PR also fixes a small issue with the styles being applied [here](https://github.com/comountainclimber/rn-task-manager/compare/feature/adds-custom-font?expand=1#diff-9a1a1daee605ac6dbead0cf66ee69bc1R74) it looks as though they might have accidentally gotten deleted during a previous refactor.

Screen Shot 2020-02-22 at 9 21 45 AM

Screen Shot 2020-02-22 at 9 21 52 AM

### CODE REVIEW

File: components/AnimatedTaskCard.js [hunk 1]
```diff
@@ -76,8 +76,7 @@ const styles = StyleSheet.create({
   taskCompletedIcon: {
     marginTop: 6
   },
-  moveText: {
-    color: "white",
-    marginLeft: 24
```
Comment (comountainclimber) [inline components/AnimatedTaskCard.js]: This was left over from refactor shuffling around and is no longer being used in this file

File: components/TaskInput.js [hunk 2]
```diff
@@ -37,9 +38,12 @@ TaskInput.propTypes = {
 };
 
 const styles = StyleSheet.create({
-  taskInput: {
+  taskInputContainer: {
```
Comment (comountainclimber) [inline components/TaskInput.js]: Seemed like this was more semantically appropriate

**Comment by comountainclimber** [comment]
@priyadarshy @bone-saw @brunobernardino 👋 tagging you guys so you can see what I have been up to. Will continue to make PRs this weekend against the issues I currently have created in this repo

**Comment by priyadarshy** [comment]
Thanks for looping us in! Using Nunito as an alternative is a great choice.

File: App.js [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+      "Nunito-Regular": require("./assets/fonts/Nunito-Regular.ttf"),
+      "Nunito-Bold": require("./assets/fonts/Nunito-Bold.ttf"),
+      "Nunito-Black": require("./assets/fonts/Nunito-Black.ttf")
+    });
+    this.setState({ fontLoaded: true });
+  }
+
+  render() {
+    const { fontLoaded } = this.state;
+    return fontLoaded && <Main />;
```
Comment (BrunoBernardino) [inline App.js]: The only thing I'd tweak/recommend would be a simple "loading" image/screen so the user isn't leeft wondering if the app has crashed, but you don't need to implement it for this demo.

**Comment by comountainclimber** [comment]
Thanks for the review @BrunoBernardino ! I went ahead and created a new issue for this problem here: https://github.com/comountainclimber/rn-task-manager/issues/8

Will tackle this later today 👍

**Pull request merged**